invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less

```
@@ -46,6 +46,17 @@ nav {
   padding-bottom: 8px;
 }
 
+@media only screen and (min-width: 745px) and (max-width: 1000px){
```

I tested with `1200px` and it looks like is fixes the issues:
```suggestion
@media only screen and (min-width: 745px) and (max-width: 1200px){
```
WDYT?